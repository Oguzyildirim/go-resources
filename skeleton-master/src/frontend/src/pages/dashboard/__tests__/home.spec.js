import { mount } from '@vue/test-utils'
import Vuex from 'vuex'
import VueRouter from 'vue-router'
import Page from '../home'

describe('home.vue', () => {
  let store

  beforeEach(() => {
    const actions = {
      'page/title': jest.fn(),
      'loading/start': jest.fn(),
      'loading/finish': jest.fn()
    }
    store = new Vuex.Store({
      actions,
      state: {
        loading: { status: false }
      }
    })
  })

  it('should match snapshot', () => {
    const router = new VueRouter({})
    const w = mount(Page, {
      store,
      router,
      stubs: ['router-link', 'router-view'],
      created () {
        this.$vuetify.lang = {
          t: () => {}
        }
        this.$vuetify.breakpoint = {
          width: 0,
          height: 0
        }
        this.$vuetify.theme = { dark: false }
      }
    })

    expect(w.html()).toMatchSnapshot()
    w.destroy()
  })
})
