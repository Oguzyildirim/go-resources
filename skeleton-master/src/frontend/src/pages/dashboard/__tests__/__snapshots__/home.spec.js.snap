// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home.vue should match snapshot 1`] = `
<section class="container container--fluid" id="dashboard">
  <div class="row">
    <div class="col-lg-4 col-12">
      <div class="v-card--material pa-3 v-card--material-chart v-card v-sheet theme--light v-card--material--has-heading" hover-reveal="">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 pa-7" style="width: 100%; background-color: rgb(233, 30, 99); border-color: #e91e63;">
            <div class="ct-square" style="max-height: 150px;"></div>
            <!---->
            <!---->
          </div>
          <!---->
        </div>
        <h4 class="card-title font-weight-light mt-2 ml-2">
          Website Views
        </h4>
        <p class="d-inline-flex font-weight-light ml-2 mt-1">
          Last Campaign Performance
        </p>
        <hr role="separator" aria-orientation="horizontal" class="mt-2 v-divider theme--light">
        <div class="v-card__actions pb-0"><i aria-hidden="true" class="v-icon notranslate mr-1 mdi mdi-clock-outline theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">updated 10 minutes ago</span></div>
      </div>
    </div>
    <div class="col-lg-4 col-12">
      <div class="v-card--material pa-3 v-card--material-chart v-card v-sheet theme--light v-card--material--has-heading" hover-reveal="">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 success pa-7" style="width: 100%;">
            <div class="ct-square" style="max-height: 150px;"></div>
            <!---->
            <!---->
          </div>
          <!---->
        </div>
        <h4 class="card-title font-weight-light mt-2 ml-2">
          Daily Sales
        </h4>
        <p class="d-inline-flex font-weight-light ml-2 mt-1"><i aria-hidden="true" class="v-icon notranslate mdi mdi-arrow-up theme--light green--text" style="font-size: 16px;"></i> <span class="green--text">55%</span>&nbsp;
          increase in today's sales
        </p>
        <hr role="separator" aria-orientation="horizontal" class="mt-2 v-divider theme--light">
        <div class="v-card__actions pb-0"><i aria-hidden="true" class="v-icon notranslate mr-1 mdi mdi-clock-outline theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">updated 4 minutes ago</span></div>
      </div>
    </div>
    <div class="col-lg-4 col-12">
      <div class="v-card--material pa-3 v-card--material-chart v-card v-sheet theme--light v-card--material--has-heading" hover-reveal="">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 info pa-7" style="width: 100%;">
            <div class="ct-square" style="max-height: 150px;"></div>
            <!---->
            <!---->
          </div>
          <!---->
        </div>
        <h3 class="card-title font-weight-light mt-2 ml-2">
          Completed Tasks
        </h3>
        <p class="d-inline-flex font-weight-light ml-2 mt-1">
          Last Last Campaign Performance
        </p>
        <hr role="separator" aria-orientation="horizontal" class="mt-2 v-divider theme--light">
        <div class="v-card__actions pb-0"><i aria-hidden="true" class="v-icon notranslate mr-1 mdi mdi-clock-outline theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">campaign sent 26 minutes ago</span></div>
      </div>
    </div>
    <div class="col-sm-6 col-lg-3 col-12">
      <div class="v-card--material pa-3 v-card--material-stats v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 info pa-7" style="max-height: 90px; width: auto;"><i aria-hidden="true" class="v-icon notranslate mdi mdi-twitter theme--dark" style="font-size: 32px;"></i>
            <!---->
            <!---->
          </div>
          <div class="ml-6">
            <div class="ml-auto text-right">
              <div class="body-3 grey--text font-weight-light">Followers</div>
              <h3 class="display-2 font-weight-light text--primary">
                +245 <small></small></h3>
            </div>
          </div>
        </div>
        <div class="px-0 col col-12">
          <hr role="separator" aria-orientation="horizontal" class="v-divider theme--light">
        </div> <i aria-hidden="true" class="v-icon notranslate ml-2 mr-1 mdi mdi-clock theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">Just Updated</span>
        <!---->
      </div>
    </div>
    <div class="col-sm-6 col-lg-3 col-12">
      <div class="v-card--material pa-3 v-card--material-stats v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 primary pa-7" style="max-height: 90px; width: auto;"><i aria-hidden="true" class="v-icon notranslate mdi mdi-poll theme--dark" style="font-size: 32px;"></i>
            <!---->
            <!---->
          </div>
          <div class="ml-6">
            <div class="ml-auto text-right">
              <div class="body-3 grey--text font-weight-light">Website Visits</div>
              <h3 class="display-2 font-weight-light text--primary">
                75.521 <small></small></h3>
            </div>
          </div>
        </div>
        <div class="px-0 col col-12">
          <hr role="separator" aria-orientation="horizontal" class="v-divider theme--light">
        </div> <i aria-hidden="true" class="v-icon notranslate ml-2 mr-1 mdi mdi-tag theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">Tracked from Google Analytics</span>
        <!---->
      </div>
    </div>
    <div class="col-sm-6 col-lg-3 col-12">
      <div class="v-card--material pa-3 v-card--material-stats v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 success pa-7" style="max-height: 90px; width: auto;"><i aria-hidden="true" class="v-icon notranslate mdi mdi-store theme--dark" style="font-size: 32px;"></i>
            <!---->
            <!---->
          </div>
          <div class="ml-6">
            <div class="ml-auto text-right">
              <div class="body-3 grey--text font-weight-light">Revenue</div>
              <h3 class="display-2 font-weight-light text--primary">
                $ 34,245 <small></small></h3>
            </div>
          </div>
        </div>
        <div class="px-0 col col-12">
          <hr role="separator" aria-orientation="horizontal" class="v-divider theme--light">
        </div> <i aria-hidden="true" class="v-icon notranslate ml-2 mr-1 mdi mdi-calendar theme--light" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">Last 24 Hours</span>
        <!---->
      </div>
    </div>
    <div class="col-sm-6 col-lg-3 col-12">
      <div class="v-card--material pa-3 v-card--material-stats v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 orange pa-7" style="max-height: 90px; width: auto;"><i aria-hidden="true" class="v-icon notranslate mdi mdi-sofa theme--dark" style="font-size: 32px;"></i>
            <!---->
            <!---->
          </div>
          <div class="ml-6">
            <div class="ml-auto text-right">
              <div class="body-3 grey--text font-weight-light">Bookings</div>
              <h3 class="display-2 font-weight-light text--primary">
                184 <small></small></h3>
            </div>
          </div>
        </div>
        <div class="px-0 col col-12">
          <hr role="separator" aria-orientation="horizontal" class="v-divider theme--light">
        </div> <i aria-hidden="true" class="v-icon notranslate ml-2 mr-1 mdi mdi-alert theme--light red--text" style="font-size: 16px;"></i> <span class="caption grey--text font-weight-light">Get More Space...</span>
        <!---->
      </div>
    </div>
    <div class="col-md-6 col-12">
      <div class="v-card--material pa-3 px-5 py-3 v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 warning pa-7" style="width: 100%;">
            <div class="display-2 font-weight-light">
              Employees Stats
            </div>
            <div class="subtitle-1 font-weight-light">
              New employees on 15th September, 2016
            </div>
            <!---->
            <!---->
          </div>
          <!---->
        </div>
        <div class="v-card__text">
          <div class="v-data-table theme--light">
            <div class="v-data-table__wrapper">
              <table>
                <colgroup>
                  <col class="">
                  <col class="">
                  <col class="">
                  <col class="">
                  <col class="">
                </colgroup>
                <thead class="v-data-table-header v-data-table-header-mobile">
                  <tr>
                    <th>
                      <div class="v-data-table-header-mobile__wrapper"></div>
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr class="v-data-table__mobile-table-row">
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">ID</div>
                      <div class="v-data-table__mobile-row__cell">1</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Name</div>
                      <div class="v-data-table__mobile-row__cell">Dakota Rice</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Salary</div>
                      <div class="v-data-table__mobile-row__cell">$35,738</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Country</div>
                      <div class="v-data-table__mobile-row__cell">Niger</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">City</div>
                      <div class="v-data-table__mobile-row__cell">Oud-Tunrhout</div>
                    </td>
                  </tr>
                  <tr class="v-data-table__mobile-table-row">
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">ID</div>
                      <div class="v-data-table__mobile-row__cell">2</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Name</div>
                      <div class="v-data-table__mobile-row__cell">Minerva Hooper</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Salary</div>
                      <div class="v-data-table__mobile-row__cell">$23,738</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Country</div>
                      <div class="v-data-table__mobile-row__cell">Curaçao</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">City</div>
                      <div class="v-data-table__mobile-row__cell">Sinaai-Waas</div>
                    </td>
                  </tr>
                  <tr class="v-data-table__mobile-table-row">
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">ID</div>
                      <div class="v-data-table__mobile-row__cell">3</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Name</div>
                      <div class="v-data-table__mobile-row__cell">Sage Rodriguez</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Salary</div>
                      <div class="v-data-table__mobile-row__cell">$56,142</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Country</div>
                      <div class="v-data-table__mobile-row__cell">Netherlands</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">City</div>
                      <div class="v-data-table__mobile-row__cell">Overland Park</div>
                    </td>
                  </tr>
                  <tr class="v-data-table__mobile-table-row">
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">ID</div>
                      <div class="v-data-table__mobile-row__cell">4</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Name</div>
                      <div class="v-data-table__mobile-row__cell">Philip Chanley</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Salary</div>
                      <div class="v-data-table__mobile-row__cell">$38,735</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Country</div>
                      <div class="v-data-table__mobile-row__cell">Korea, South</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">City</div>
                      <div class="v-data-table__mobile-row__cell">Gloucester</div>
                    </td>
                  </tr>
                  <tr class="v-data-table__mobile-table-row">
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">ID</div>
                      <div class="v-data-table__mobile-row__cell">5</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Name</div>
                      <div class="v-data-table__mobile-row__cell">Doris Greene</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Salary</div>
                      <div class="v-data-table__mobile-row__cell">$63,542</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">Country</div>
                      <div class="v-data-table__mobile-row__cell">Malawi</div>
                    </td>
                    <td class="v-data-table__mobile-row">
                      <div class="v-data-table__mobile-row__header">City</div>
                      <div class="v-data-table__mobile-row__cell">Feldkirchen in Kārnten</div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="v-data-footer">
              <div class="v-data-footer__select">
                <div class="v-input v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light v-text-field v-select">
                  <div class="v-input__control">
                    <div role="button" aria-haspopup="listbox" aria-expanded="false" aria-owns="list-62" class="v-input__slot">
                      <div class="v-select__slot">
                        <div class="v-select__selections">
                          <div class="v-select__selection v-select__selection--comma">10</div><input aria-label="$vuetify.dataTable.itemsPerPageText" id="input-62" readonly="readonly" type="text" aria-readonly="false" autocomplete="off">
                        </div>
                        <div class="v-input__append-inner">
                          <div class="v-input__icon v-input__icon--append"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light">$dropdown</i></div>
                        </div><input type="hidden" value="10">
                      </div>
                      <div class="v-menu">
                        <!---->
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div class="v-data-footer__pagination"></div>
              <div class="v-data-footer__icons-before"><button type="button" disabled="disabled" class="v-btn v-btn--disabled v-btn--flat v-btn--icon v-btn--round v-btn--text theme--light v-size--default"><span class="v-btn__content"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light">$prev</i></span></button></div>
              <div class="v-data-footer__icons-after"><button type="button" disabled="disabled" class="v-btn v-btn--disabled v-btn--flat v-btn--icon v-btn--round v-btn--text theme--light v-size--default"><span class="v-btn__content"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light">$next</i></span></button></div>
            </div>
          </div>
        </div>
        <!---->
      </div>
    </div>
    <div class="col-md-6 col-12">
      <div class="v-card--material pa-3 px-5 py-3 v-card v-sheet theme--light v-card--material--has-heading">
        <div class="d-flex grow flex-wrap">
          <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 info pa-7" style="width: 100%;">
            <div class="v-tabs theme--dark">
              <div role="tablist" class="v-item-group theme--dark v-slide-group v-tabs-bar white--text transparent">
                <div class="v-slide-group__prev v-slide-group__prev--disabled">
                  <transition-stub name="fade-transition"></transition-stub>
                </div>
                <div class="v-slide-group__wrapper">
                  <div class="v-slide-group__content v-tabs-bar__content">
                    <div class="v-tabs-slider-wrapper" style="transition: none;">
                      <div class="v-tabs-slider white"></div>
                    </div><span class="subheading font-weight-light mx-3" style="align-self: center;">Tasks:</span>
                    <div tabindex="0" aria-selected="true" role="tab" class="mr-3 v-tab v-tab--active"><i aria-hidden="true" class="v-icon notranslate mr-2 mdi mdi-bug theme--dark"></i>
                      Bugs
                    </div>
                    <div tabindex="0" aria-selected="false" role="tab" class="mr-3 v-tab"><i aria-hidden="true" class="v-icon notranslate mr-2 mdi mdi-code-tags theme--dark"></i>
                      Website
                    </div>
                    <div tabindex="0" aria-selected="false" role="tab" class="v-tab"><i aria-hidden="true" class="v-icon notranslate mr-2 mdi mdi-cloud theme--dark"></i>
                      Server
                    </div>
                  </div>
                </div>
                <div class="v-slide-group__next v-slide-group__next--disabled">
                  <transition-stub name="fade-transition"></transition-stub>
                </div>
              </div>
            </div>
            <!---->
            <!---->
          </div>
          <!---->
        </div>
        <div class="v-window transparent v-item-group theme--light v-tabs-items">
          <div class="v-window__container">
            <transition-stub name="">
              <div class="v-window-item v-window-item--active">
                <div class="v-card__text">
                  <div class="row align-center">
                    <div class="col col-1">
                      <div class="v-list-item__action">
                        <div class="v-input v-input--is-label-active v-input--is-dirty theme--light v-input--selection-controls v-input--checkbox secondary--text">
                          <div class="v-input__control">
                            <div class="v-input__slot">
                              <div class="v-input--selection-controls__input"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light secondary--text">$checkboxOn</i><input aria-checked="true" id="input-86" role="checkbox" type="checkbox" value="">
                                <div class="v-input--selection-controls__ripple secondary--text"></div>
                              </div>
                            </div>
                            <div class="v-messages theme--light secondary--text">
                              <transition-group-stub tag="div" name="message-transition" class="v-messages__wrapper"></transition-group-stub>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div class="col col-9">
                      <div class="font-weight-light">Sign contract for "What are conference organizers afraid of?"</div>
                    </div>
                    <div class="text-right col col-2"><i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-pencil theme--light"></i> <i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-close theme--light error--text"></i></div>
                  </div>
                  <div class="row align-center">
                    <div class="col col-1">
                      <div class="v-list-item__action">
                        <div class="v-input theme--light v-input--selection-controls v-input--checkbox">
                          <div class="v-input__control">
                            <div class="v-input__slot">
                              <div class="v-input--selection-controls__input"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light">$checkboxOff</i><input aria-checked="false" id="input-92" role="checkbox" type="checkbox" value="">
                                <div class="v-input--selection-controls__ripple"></div>
                              </div>
                            </div>
                            <div class="v-messages theme--light">
                              <transition-group-stub tag="div" name="message-transition" class="v-messages__wrapper"></transition-group-stub>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div class="col col-9">
                      <div class="font-weight-light">Lines From Great Russian Literature? Or E-mails From My Boss?</div>
                    </div>
                    <div class="text-right col col-2"><i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-pencil theme--light"></i> <i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-close theme--light error--text"></i></div>
                  </div>
                  <div class="row align-center">
                    <div class="col col-1">
                      <div class="v-list-item__action">
                        <div class="v-input theme--light v-input--selection-controls v-input--checkbox">
                          <div class="v-input__control">
                            <div class="v-input__slot">
                              <div class="v-input--selection-controls__input"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light">$checkboxOff</i><input aria-checked="false" id="input-98" role="checkbox" type="checkbox" value="">
                                <div class="v-input--selection-controls__ripple"></div>
                              </div>
                            </div>
                            <div class="v-messages theme--light">
                              <transition-group-stub tag="div" name="message-transition" class="v-messages__wrapper"></transition-group-stub>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div class="col col-9">
                      <div class="font-weight-light">Flooded: One year later, assessing what was lost and what was found when a ravaging rain swept through metro Detroit</div>
                    </div>
                    <div class="text-right col col-2"><i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-pencil theme--light"></i> <i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-close theme--light error--text"></i></div>
                  </div>
                  <div class="row align-center">
                    <div class="col col-1">
                      <div class="v-list-item__action">
                        <div class="v-input v-input--is-label-active v-input--is-dirty theme--light v-input--selection-controls v-input--checkbox secondary--text">
                          <div class="v-input__control">
                            <div class="v-input__slot">
                              <div class="v-input--selection-controls__input"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light secondary--text">$checkboxOn</i><input aria-checked="true" id="input-104" role="checkbox" type="checkbox" value="">
                                <div class="v-input--selection-controls__ripple secondary--text"></div>
                              </div>
                            </div>
                            <div class="v-messages theme--light secondary--text">
                              <transition-group-stub tag="div" name="message-transition" class="v-messages__wrapper"></transition-group-stub>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div class="col col-9">
                      <div class="font-weight-light">Create 4 Invisible User Experiences you Never Knew About</div>
                    </div>
                    <div class="text-right col col-2"><i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-pencil theme--light"></i> <i aria-hidden="true" class="v-icon notranslate mx-1 mdi mdi-close theme--light error--text"></i></div>
                  </div>
                </div>
              </div>
            </transition-stub>
            <transition-stub name="">
              <!---->
            </transition-stub>
            <transition-stub name="">
              <!---->
            </transition-stub>
          </div>
        </div>
        <!---->
      </div>
    </div>
  </div>
</section>
`;
